tests/Sinch.Tests/e2e/Verification/ReportVerificationTests.cs

```diff
@@ -54,26 +50,20 @@ public async Task ReportFlashCallByIdentity()
                     },
                 });
 
-            response.Should().BeOfType<ReportFlashCallVerificationResponse>().Which.Should().BeEquivalentTo(
+            response.Should().BeEquivalentTo(
                 new ReportFlashCallVerificationResponse()
                 {
-                    Method = VerificationMethod.FlashCall,
-                    Id = "_id",
-                    Price = new Price()
+                    Id = "123456",
+                    Status = VerificationStatus.Fail,
+                    Reason = Reason.DestinationDenied,
+                    Reference = "ref",
+                    Source = Source.Manual,
+                    Identity = new Identity()
                     {
-                        VerificationPrice = new PriceDetail()
-                        {
-                            Amount = 0.42,
-                            CurrencyId = "EUR"
-                        },
-                        TerminationPrice = new PriceDetail()
-                        {
-                            Amount = 0.11,
-                            CurrencyId = "EUR"
-                        },
-                        BillableDuration = 40
+                        Type = IdentityType.Number,
```

Comment:
Not related to this PR, just a thought about Identity.
Currently only number is supported.
What do you think about creating something like a dedicated `NumberIdentity` class to simplify interface with this type ?
e.g. for StartVerification request:
```
var identity = new NumberIdentity("+123456789");
var request = .... <identity>
```
It will simplify type usage, hide the protocol dedicated value and in case of new identity type will be supported, it won't cause any breaking change

Reply:
yeah, makes sense, I'll add that too after release